2197/1/koans/triangle.rb

```diff
@@ -0,0 +1,36 @@
+# rubocop:disable all
+# Triangle Project Code.
+
+# Triangle analyzes the lengths of the sides of a triangle
+# (represented by a, b and c) and returns the type of triangle.
+#
+# It returns:
+#   :equilateral  if all sides are equal
+#   :isosceles    if exactly 2 sides are equal
+#   :scalene      if no sides are equal
+#
+# The tests for this method can be found in
+#   about_triangle_project.rb
+# and
+#   about_triangle_project_2.rb
+#
+# :reek:FeatureEnvy
+# :reek:UncommunicativeParameterName
+def triangle(a, b, c)
+
+	if a < 0
+		raise TriangleError
+	elsif a + b <= c or a + c <= b or b + c <= a
```

Comment:
read this and change code for the method:
- https://github.com/rubocop-hq/ruby-style-guide